vnext/Microsoft.ReactNative/Fabric/ComponentViewRegistry.h

```diff
@@ -0,0 +1,38 @@
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include "ComponentView.h"
+
+namespace Microsoft::ReactNative {
+
+struct ComponentViewDescriptor final {
+  std::shared_ptr<IComponentView> view{nullptr};
+
+  /*
+   * Indicates a requirement to call on the view methods from
+   * `RCTMountingTransactionObserving` protocol.
```

Comment:
is this documented somewhere / can you plop a link to it in the comment?

Reply:
This is refering to some stuff in core that isn't implemented yet, so I'll remove this for now. 

But to answer your question:
https://github.com/facebook/react-native/blob/6e6443afd04a847ef23fb6254a84e48c70b45896/React/Fabric/Mounting/RCTMountingTransactionObserving.h